assets/moonhub.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- type: application
```

Comment:
Should say it depends on Cohere
https://twitter.com/aidangomez/status/1712233505067803090